frontend/src/views/RoleView.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+  <div class="PageWrapper">
+    <div class="TableContainer">
+      <h1>{{ roleName }}</h1>
+      <h2>Attached Knowledges</h2>
+      <br />
+      <ElTable :data="knowledges" style="width: 100%">
+        <ElTableColumn prop="name" label="Knowledges" />
+        <ElTableColumn fixed="right" label="Actions" width="150">
+          <template #default="{ row }">
+            <div>
+              <ElButton @click="handleDetail(row)" type="text" size="small">Delete</ElButton>
```

Comment:
```suggestion
              <ElButton @click="onDeleteRow(row)" type="text" size="small">Delete</ElButton>
```

Pay attention the names you are using